fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -112,6 +155,70 @@ def sst_dst_to_dense(self, sst: Tensor, dst: Tensor = None) -> Tensor:
         """
         pass
 
+    def _transform(self, dense: Tensor) -> Tensor:
+        """
+        Applies the transformation algorithm chosen by the caller to the input dense tensor.
+        Returns the transformed tensor.
+
+        Args:
+            dense (Tensor):
+                Input dense tensor.
+        """
+        if self._algo is Algo.FFT:
+            transform = torch.fft.fft  # type: ignore
```

Comment:
if this function is just a one line call, why not do that during init?

I.e. in the init:

self._transform = torch.fft.fft if algo == FFT else _dct_placeholder

Then, you don't need this one line function. Just call self._transform directly would be OK? You can define a _dct_placeholder function just to raise the NotImplementedError(), right?